src/Core/MainLoop.js

```diff
@@ -68,7 +68,19 @@ function updateElements(context, geometryLayer, elements) {
         // update attached layers
         for (const attachedLayer of geometryLayer._attachedLayers) {
             if (attachedLayer.ready) {
-                attachedLayer.update(context, attachedLayer, element);
+                // Attached layers expect to receive the visual representation
```

Comment:
An example would be needed to understand the use case.
it's necessary to add a new update?

Reply:
You can find an example in the https://github.com/peppsac/itowns2/tree/colorisation branch.

It uses the ColorTextureProcessing to colorize a 3dtiles pointcloud.

But 3dTilesProcessing update function process metadata objects (= the content of the tileset.json file) while ColorTextureProcessing expects objects with a material (because it was initially built for TileMesh).